egs2/chime2/enh1/local/reverb_wsj0_data_prep.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
```

Comment:
Sorry, I just took a rough look at this file and the previous one `egs2/chime2/enh1/local/noisy_wsj0_data_prep.sh`. Are there big differences? If so, just ignore this comment. If not, can those two files be combined?

Reply:
Yes, I think they are quite different. Again, I just copied them from Kaldi's chime2 recipe.